kiosk-helm/Chart.yaml

```diff
@@ -0,0 +1,27 @@
+# SPDX-License-Identifier: Apache-2.0
+#!BuildTag: suse/kiosk/kiosk-helm:latest
+#!BuildTag: suse/kiosk/kiosk-helm:1.0.13
+apiVersion: v2
+name: kiosk
+description: Manage kiosk or HID workloads using Kubernetes
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 1.0.0
```

Comment:
should be 1.0.13

Reply:
I'm fine with 1.0.0 but we should be "icnremental" to whatever we had previously

Reply:
there's no previous one, unless you count Andrew's openSUSE version, which was 0.1.0.